docs/architecture/discovery/forms-data-logical-data-model.md

```diff
@@ -0,0 +1,590 @@
+- [Reprocessor accreditation](https://forms-runner.test.cdp-int.defra.cloud/form/preview/draft/demo-for-pepr-extended-producer-responsibilities-apply-for-accreditation-as-a-packaging-waste-reprocessor-ea/form-guidance)
+
+## Logical Data Model (LDM)
+
+```mermaid
+erDiagram
+  ORGANISATION {
+    ObjectId _id PK "_id from organisation collection"
+    int orgId "org id from organisation collection"
+    enum status "created,approved,rejected,suspended,archived"
+    STATUS-HISTORY[] statusHistory "Status change history"
+    int schemaVersion
+    int version "Version of the document. incremented during write and can be used for optimistic locking in future"
+    enum[] wasteProcessingTypes "reprocessor, exporter"
+    enum[] reprocessingNations "england,wales, scotland,northern_ireland"
+    enum businessType "individual, unincorporated, partnership"
+    COMPANY-DETAILS companyDetails
+    PARTNERSHIP partnership "company partnership details"
+    USER contactDetails "organisation application contact details"
+    REGISTRATION[] registrations
+    ACCREDITATION[] accreditations
+    ISO8601 formSubmissionTime
+  }
+
+
+  REGISTRATION {
+    ObjectId id  PK "_id from registration collection"
+    ISO8601 formSubmissionTime
+    enum status "created,approved,rejected,suspended,archived"
+    STATUS-HISTORY[] statusHistory "Status change history"
+    SITE site "applicable only for reprocessor"
+    enum material "aluminium,fibre,glass,paper,plastic,steel,wood"
+    enum wasteProcessingType "reprocessor, exporter. This might need to be parsed from form name or another field"
+    ObjectId accreditationId FK "automatically linked when possible"
+    string gridReference "applicable only for reprocessor"
+    enum[] recyclingProcess "glass_re_melt,glass_other"
+    ADDRESS noticeAddress
+    string wasteRegistrationNumber "waste carrier, broker or dealer registration number"
+    WASTE-MANAGEMENT-PERMIT[] wasteManagementPermits "permit or waste exemption details"
+    USER[] approvedPersons "approved persons for this site"
+    string suppliers "Waste supplier details. applicable only for exporter"
+    string[] exportPorts "List of ports from which waste is exported."
+    YEARLY-METRICS[] yearlyMetrics "Yearly metrics related to how much input is received, output produced etc..."
+    string plantEquipmentDetails "a list of the key plant and equipment used for reprocessing. applicable only for reprocessor"
+    USER contactDetails "contact details for registration"
+    USER submitterContactDetails "who submitted the application"
+    string[] samplingInspectionPlan "sampling and inspection plan documents"
+    string[] overseasSites "overseas reprocessing and interim sites log documents"
+  }
+
+
+  STATUS-HISTORY{
+    enum status "created,approved,rejected,suspended,archived"
+    ISO8601 changedAt "At what time the status changed"
```

Comment:
can we use `updatedAt` and `updatedBy`? I think this will keep things consistent with the [Waste Record in the LLD](https://github.com/DEFRA/epr-backend/blob/a16a8e1d4735393446634b8f7619135930afe11d/docs/architecture/discovery/pepr-lld.md?plain=1#L274-L275) and also give us the ability to surface information about who changed the status.